src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2751,6 +2752,28 @@
           "$ref": "#/definitions/translationMetadata"
         },
 
+        "supportedTaxonomies": {
+          "description": "An array of toolComponentReference objects to declare the taxonomies supported by the tool component.",
+          "type": "array",
+          "minItems": 0,
+          "uniqueItems": true,
+          "default": [],
+          "items": {
+            "$ref": "#/definitions/toolComponentReference"
+          }
+        },
+
+        "reportingConfigurationOverrides": {
```

Comment:
>reportingConfigurationOverrides [](start = 9, length = 31)

I think we should remove this. reportingConfigurationOverrides are now provided by 'policy' objects that simply recapitulate the core toolComponent and make changes to its reporting descriptor defaultConfiguration values.